### DISCUSSION TITLE
TOC and TOD

### DISCUSSION THREAD

**Discussion by Lewis2023** [albar965/littlenavmap#991] [Q&A]
For dead reckoning it would be fine to have the TOC and TOD in the flightplan to check distance and time during flight.
Or, perhaps I have overseen this feature.
Thanks for the last update and for this exceptional app.

**Comment by albar965** [comment]
I have this already in mind but it is a huge effort to change this in the plan table (all indexes, menus and whatever have to be changed). So it is unlikely to happen soon, sorry.

**Reply by Lewis2023** [reply]
Thanks for your answer. In the meantime I set a waypoint near to the TOC as a starting point and reference for dead reckoning.
